bb.edn

```diff
@@ -22,7 +22,9 @@
          
          ffix {:doc "Format source files" 
                :task (clojure "-M:ffix")}
-
+         
+         dupgrade {:doc "Upgrade all clojure deps"
+                   :task (clojure "-M:outdated" "--upgrade" "--force")}
```

Comment:
We could also have this without `--force`, then each package upgrade has to be confirmed